lint/action.yaml

```diff
@@ -33,37 +33,6 @@ runs:
         fetch-depth: 0
     - name: Setup tools
       uses: open-turo/action-setup-tools@v1
-    - name: Check for yarn.lock
-      id: check_yarn_lock
-      uses: andstor/file-existence-action@v3
-      with:
-        files: yarn.lock
-    - name: Install dependencies (yarn)
-      if: steps.check_yarn_lock.outputs.files_exists == 'true'
-      shell: bash
-      run: yarn --pure-lockfile
-      env:
-        NPM_AUTH_TOKEN: ${{ inputs.npm-auth-token }}
-    - name: Install dependencies (npm)
-      if: steps.check_yarn_lock.outputs.files_exists == 'false'
-      shell: bash
-      run: npm ci
-      env:
-        NPM_AUTH_TOKEN: ${{ inputs.npm-auth-token }}
```

Comment:
Wouldn't we need to still install dependencies so that pre-commit hooks like eslint work?